### ISSUE TITLE
Sails.js in production mode

### ISSUE THREAD

**Issue by mhipo1364** [balderdashy/sails#3143]
Hi guys

I need help and advice

I'm developing my project with `sails.js`, I need to add load balancer to my project(by using `NginX`), how can I run multiple app by `sails.js` and add it on `NginX` upstream?

**Comment by jimlei** [comment]
You could use [forever](https://github.com/foreverjs/forever) or [pm2](https://github.com/Unitech/pm2) to launch / run multiple instances

PM2 example that runs one instance per core and uses nodes built in load balancer. Note that you should have Node >= 0.11 to use cluster as it was experimental in 0.10

```json
{
  "apps" : [
    {
      "name"               : "yourappname",
      "script"             : "app.js",
      "instances"          : "0",
      "merge_logs"         : true,
      "exec_mode"          : "cluster",
      "max_memory_restart" : "512M",
      "autorestart"        : true,
      "error_file"         : "./logs/error.log",
      "out_file"           : "./logs/out.log",
      "pid_file"           : "./logs/app.pid",
      "log_date_format"    : "YYYY-MM-DD HH:mm:ss Z",
    }
  ]
}
```

```sh
$ pm2 start processes.json
```

**Comment by mhipo1364** [comment]
@jimlei : Thanks, but I want to run it over `NginX` web server, and in `NginX` i want to declare `upstream` to proxy several ports in port 80, I just wonder how can I runt my project on several ports.

**Comment by Geocloud** [comment]
You can use this article to follow these steps: How to use Nginx with Salsas
https://www.digitalocean.com/community/tutorials/how-to-create-an-node-js-app-using-sails-js-on-an-ubuntu-vps

**Comment by mhipo1364** [comment]
Thanks @Geocloud :+1:

**Comment by sailsbot** [comment]
Thanks for posting, @mhipo1364.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by jasalazar** [comment]
Hello guys.

Is this guide updated? Since i read the following 

> Open up config/application.js

But there is not such file on Sails 0.11.3 version.

Greetings!
